### PR TITLE
use check/trigger uuid instead of id

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: alamo_alerter/alerter.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def __init__(self, driver, notification, statsd):
         self.statsd = statsd
 
     @stats.timer()
-    def get_cached_result(self, check_id: int, trigger_id: int, debounce: int):
+    def get_cached_result(self,
+                          check_uuid: str, trigger_uuid: str, debounce: int):
```
Comment (deejay1) [inline alamo_alerter/alerter.py]: str? not an uuid instance?

Comment (pnuckowski) [inline alamo_alerter/alerter.py]: Here is still str. Only in `driver` uuid are transformed into UUID instances.

**Comment by deejay1** [comment]
:+1:

**Comment by benethor** [comment]
:+1:

**Pull request merged**